_sass/blocks/revenues/_revenues_subpage_diving_graphics.scss

```diff
@@ -310,9 +309,21 @@
       background: url('../img/revenues/minerals-wide.png') no-repeat;
       background-position-x: -1000px;
       background-size: 222%;
+
       &:before {
         background: none;
       }
     }
+
+    background: $minerals-primary-color;
```

Comment:
Expected item on line 318 to appear before line 273. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets